nearcore/src/config.rs

```diff
@@ -790,8 +790,10 @@ pub fn init_configs(
     num_shards: NumShards,
     fast: bool,
     genesis: Option<&str>,
-    download: bool,
+    should_download_genesis: bool,
```

Comment:
Not sure we need `should_` prefix, for me `download_genesis` is enough

Reply:
It is due to a conflict with a function with the same name. It was complaining.

Reply:
```
sandi@sandi-ThinkPad-X1-Carbon-7th ~/w/n/nearcore (download-config)> vim nearcore/src/config.rs
sandi@sandi-ThinkPad-X1-Carbon-7th ~/w/n/nearcore (download-config)> git diff
diff --git a/nearcore/src/config.rs b/nearcore/src/config.rs
index 3f0efe36..465843bf 100644
--- a/nearcore/src/config.rs
+++ b/nearcore/src/config.rs
@@ -792,7 +792,7 @@ pub fn init_configs(
     genesis: Option<&str>,
     should_download_genesis: bool,
     download_genesis_url: Option<&str>,
-    should_download_config: bool,
+    download_config: bool,
     download_config_url: Option<&str>,
     max_gas_burnt_view: Option<Gas>,
 ) {
@@ -812,7 +812,7 @@ pub fn init_configs(
     if let Some(url) = download_config_url {
         download_config(&url.to_string(), &dir.join(CONFIG_FILENAME));
         config = Config::from_file(&dir.join(CONFIG_FILENAME));
-    } else if should_download_config {
+    } else if download_config {
         let url = get_config_url(&chain_id);
         download_config(&url, &dir.join(CONFIG_FILENAME));
         config = Config::from_file(&dir.join(CONFIG_FILENAME));
sandi@sandi-ThinkPad-X1-Carbon-7th ~/w/n/nearcore (download-config)> cargo build -p neard
   Compiling near-vm-errors v3.0.0 (/home/sandi/workspace/near/nearcore/runtime/near-vm-errors)
   Compiling near-primitives-core v0.1.0 (/home/sandi/workspace/near/nearcore/core/primitives-core)
   Compiling near-crypto v0.1.0 (/home/sandi/workspace/near/nearcore/core/crypto)
   Compiling near-primitives v0.1.0 (/home/sandi/workspace/near/nearcore/core/primitives)
   Compiling near-chain-configs v0.1.0 (/home/sandi/workspace/near/nearcore/core/chain-configs)
   Compiling near-store v2.2.0 (/home/sandi/workspace/near/nearcore/core/store)
   Compiling near-chain-primitives v0.1.0 (/home/sandi/workspace/near/nearcore/chain/chain-primitives)
   Compiling near-pool v0.1.0 (/home/sandi/workspace/near/nearcore/chain/pool)
   Compiling near-vm-logic v3.0.0 (/home/sandi/workspace/near/nearcore/runtime/near-vm-logic)
   Compiling near-vm-runner v3.0.0 (/home/sandi/workspace/near/nearcore/runtime/near-vm-runner)
   Compiling near-chain v0.1.0 (/home/sandi/workspace/near/nearcore/chain/chain)
   Compiling node-runtime v3.0.0 (/home/sandi/workspace/near/nearcore/runtime/runtime)
   Compiling near-network v0.1.0 (/home/sandi/workspace/near/nearcore/chain/network)
   Compiling near-epoch-manager v0.0.1 (/home/sandi/workspace/near/nearcore/chain/epoch_manager)
   Compiling near-chunks v0.1.0 (/home/sandi/workspace/near/nearcore/chain/chunks)
   Compiling near-client-primitives v0.1.0 (/home/sandi/workspace/near/nearcore/chain/client-primitives)
   Compiling near-jsonrpc-primitives v0.2.0 (/home/sandi/workspace/near/nearcore/chain/jsonrpc-primitives)
   Compiling near-client v0.1.0 (/home/sandi/workspace/near/nearcore/chain/client)
   Compiling near-jsonrpc-client v0.1.0 (/home/sandi/workspace/near/nearcore/chain/jsonrpc/client)
   Compiling near-jsonrpc v0.2.2 (/home/sandi/workspace/near/nearcore/chain/jsonrpc)
   Compiling nearcore v1.2.0 (/home/sandi/workspace/near/nearcore/nearcore)
error[E0618]: expected function, found `bool`
   --> nearcore/src/config.rs:813:9
    |
795 |     download_config: bool,
    |     --------------- `bool` defined here
...
813 |         download_config(&url.to_string(), &dir.join(CONFIG_FILENAME));
    |         ^^^^^^^^^^^^^^^----------------------------------------------
    |         |
    |         call expression requires function

error[E0618]: expected function, found `bool`
   --> nearcore/src/config.rs:817:9
    |
795 |     download_config: bool,
    |     --------------- `bool` defined here
...
817 |         download_config(&url, &dir.join(CONFIG_FILENAME));
    |         ^^^^^^^^^^^^^^^----------------------------------
    |         |
    |         call expression requires function

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0618`.
error: could not compile `nearcore`

To learn more, run the command again with --verbose.
```